### PR TITLE
fix: Quoting of keybding var

### PR DESCRIPTION
**Pull request by munen**
Hi @lordpretzel 

Thank you very much for this interesting library! As a prolific mu4e user, I'm very much intrigued! Just recently, I've written a blog post on how to use [mu4e with HTML mails](https://200ok.ch/posts/2020-05-27_using_emacs_and_mu4e_for_emails_even_with_html.html) - but your project is going to make that obsolete, real soon^^

I just compiled Emacs 27.1 with xwidgets support to test. And it does work! Having said that, I'm not going to use it, because xwidgets is flickering flickering a lot - and it seems that it upsets my otherwise very stable Emacs. But I'll happily keep track of the progress in this repo and of xwidgets and will come back to the future.

Ok, on to the rationale of this PR: It fixes `Debugger entered--Lisp error: (void-variable mu4e-views-mu4e-select-view-msg-method)`.

Thank you for your effort and all the best!🙏

### CODE REVIEW

**Comment by lordpretzel** [comment]
thanks! Sorry that xwidgets wasn't stable. For me to know for the future, do you experience the flickering only in mu4e-views or also when just browsing some url in xwidgets?

All the best,

    Boris

**Comment by munen** [comment]
Oh, I apologize! I didn't want to imply that the flickering was due to mu4e-views; it's a general issue that I experienced with xwidgets. Maybe some configuration of mine did interfere, I don't know. It's been the first time I've compiled with xwidgets support, though.

Let me rephrase: mu4e-views was working perfectly for me in my tests! And I'm looking forward to using it on a daily basis^^

**Comment by lordpretzel** [comment]
no need to apologize! Glad to hear that it's not mu4e-views though.
